packages/dev/core/src/Meshes/Builders/greasedLineBuilder.ts

```diff
@@ -1,14 +1,18 @@
-import type { GreasedLineMaterialOptions } from "../../Materials/greasedLinePluginMaterial";
-import { GreasedLineMeshMaterialType, GreasedLinePluginMaterial } from "../../Materials/greasedLinePluginMaterial";
 import { StandardMaterial } from "./../../Materials/standardMaterial";
 import { PBRMaterial } from "../../Materials/PBR/pbrMaterial";
 import type { Nullable } from "../../types";
-import type { GreasedLineMeshOptions } from "../greasedLineMesh";
-import { GreasedLineMesh } from "../greasedLineMesh";
+import { GreasedLineMesh } from "../GreasedLine/greasedLineMesh";

```

Comment:
I would also move the `greasedLineBuilder.ts` file to the `GreasedLine/` folder so that all files are in a single location.
But as it is a builder, I guess it's debatable: let's ask @sebavan.

Reply:
I was thinking about it as well, but in my opinion it fits better into the builders folder.
Waiting for @sebavan

Reply:
I'll push it now to see whether it passes all checks.

Reply:
I do not have any preferences, lets ask @RaananW what he thinks ?

Reply:
I might have missed a conversation - why do we need a new directory? This is the first time we move meshes to a subdirectory in the Meshes folder

I wouldn't move the builder. This way you could theoretically import the entire builder directory and use it directly. In this case the dependencies (i.e. the underlying LineMesh modules) will be automatically loaded when importing the builder.

Reply:
I suggested creating a `GreasedLine/` subdirectory in the `Materials/` and `Meshes/` folders because this feature is implemented with 3 files under `Materials/` and 4 under `Meshes/`. I thought this was better for organizational reasons, rather than having files piling up in the main directories. But maybe that's not the case!

Reply:
TBH i'm very neutral about it. Just wanted to understand the reason behind it.
Can stay this way, as long as we make sure the imports are working as expected (which they are, as far as I see)